includes/class-admin.php

```diff
@@ -390,6 +390,16 @@ public static function register_settings() {
 				'default'      => 1,
 			)
 		);
+		register_setting(
```

Comment:
I am not sure if we should add more settings like this, if we work on templating! We should keep the settings clean and maybe work on different templates, a user can choose from!

Reply:
Maybe we add a const instead, for the quick-win.

Reply:
How about a filter?

Reply:
I added a filter, and if we remove the setting, then we just leave the filter.

Reply:
I would remove the settings anyways, otherwise I think we get a lot messages if we remove them, why they are gone. Atm. there is only one person having the issue, so I would prefer using only the filter and/or a constant.

Reply:
Removed setting.